test/views/employer/cis/uk/subcontractor/WasTurnoverMoreAfterVATViewSpec.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.employer.cis.uk.subcontractor
+
+import play.api.data.Form
+import forms.employer.cis.uk.subcontractor.WasTurnoverMoreAfterVATFormProvider
+import models.employer.cis.uk.subcontractor.WasTurnoverMoreAfterVAT
+import play.twirl.api.HtmlFormat
+import views.behaviours.ViewBehaviours
+import views.html.employer.cis.uk.subcontractor.wasTurnoverMoreAfterVAT
+
+class WasTurnoverMoreAfterVATViewSpec extends ViewBehaviours {
+
+  val messageKeyPrefix = "wasTurnoverMoreAfterVAT"
+
+  val form = new WasTurnoverMoreAfterVATFormProvider()()
+
+  val serviceInfoContent = HtmlFormat.empty
+
+  def createView = () => wasTurnoverMoreAfterVAT(frontendAppConfig, form)(serviceInfoContent)(fakeRequest, messages)
+
+  def createViewUsingForm =
+    (form: Form[_]) => wasTurnoverMoreAfterVAT(frontendAppConfig, form)(serviceInfoContent)(fakeRequest, messages)
+
+  "WasTurnoverMoreAfterVAT view" must {
+    behave like normalPage(createView, messageKeyPrefix)
+  }
+
+  "WasTurnoverMoreAfterVAT view" when {
+    "rendered" must {
+      "contain radio buttons for the value" in {
+        val doc = asDocument(createViewUsingForm(form))
+        for (option <- WasTurnoverMoreAfterVAT.options) {
+          assertContainsRadioButton(doc, option.id, "value", option.value, false)
+        }
+      }
+    }
+
+    for (option <- WasTurnoverMoreAfterVAT.options) {
+      s"rendered with a value of '${option.value}'" must {
+        s"have the '${option.value}' radio button selected" in {
+          val doc = asDocument(createViewUsingForm(form.bind(Map("value" -> s"${option.value}"))))
+          assertContainsRadioButton(doc, option.id, "value", option.value, true)
+
+          for (unselectedOption <- WasTurnoverMoreAfterVAT.options.filterNot(o => o == option)) {
```

Comment:
You can change the lamba to be `_ == option` it is a little more expressive and there is less noise

Reply:
Lets see if we can modify template!!